src/components/MenuItem.tsx

```diff
@@ -268,6 +268,9 @@ type MenuItemBaseProps = {
 
     /** Optional account id if it's user avatar or policy id if it's workspace avatar */
     avatarID?: number | string;
+
+    /** Should we center text or not */
```

Comment:
```suggestion
    /** Should we center all text or not */
```

I think it's right? It looks like there is multiple text in this component that would get centered.